src/pages/ProblemEditor/ProblemEditorPage.tsx

```diff
@@ -2,8 +2,29 @@ import React from "react";
 import { ProblemEditorContainer } from "./ProblemEditorContainer/ProblemEditorContainer";
 import { LayoutContainer } from "../../shared/LayoutContainer";
 import { useAppSelector } from "../../app/common/hooks";
+import { useLocation, useParams } from "react-router-dom";
+
+interface ProblemEditorURL {
+  problemId?: string;
+}
+
+interface ProblemCreatorURL {
+  moduleId?: string;
+}
+
+interface ProblemLocationState {
+  moduleName?: string;
+}
 
 export const ProblemEditorPage = () => {
+  const { moduleId, problemId } = useParams<
+    ProblemEditorURL & ProblemCreatorURL
+  >();
+  const { moduleName } = useLocation<ProblemLocationState>().state;
+  console.log("moduleId: " + moduleId);
```

Comment:
Leaving these in so that Tanner can verify functionality easily. Make sure you delete them once you're done thought @tanner-ropp.